docs/guide/migration.md

```diff
@@ -0,0 +1,86 @@
+# Migration
+
+This page discusses migration from LangChain `GraphVectorStore` as well as between versions of `graph-retriever` and `langchain-graph-retriever`.
+
+## From LangChain GraphVectorStore
+
+LangChain `GraphVectorStore` relied on putting specially crafted `Link` instances into `metadata["links"]`. Many cases used link extractors to compute these links, but it was also often useful (and necessary) to create them manually.
+
+When converting from a `GraphVectorStore` to the new `langchain-graph-retriever` library, you need to do the following:
+
+1. Replace uses of the link extractors with document transformers.
+2. Replace manualy link creation with metadata fields.
+3. Replace `GraphVectorStore` usage with the `GraphRetriever`.
```

Comment:
Can we include a section explaining how to use existing datasets with the new library? I believe we wrote an edge extractor that could handle the links structure, and it could be helpful for people to migrate incrementally.

Reply:
We haven't written one yet, so we can't write that section. We have made it *possible* to write such  an edge extractor, but we have not done so.